cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -37,6 +38,11 @@ class Robot(BaseSalesforceTask):
         "debug": {
             "description": "If true, enable the `breakpoint` keyword to enable the robot debugger"
         },
+        "processes": {
+            "description": "*zexperimental* Number of parallel processes to use for running tests in parallel. If this value is set to a number larger than 1 the tests will run using the open source tool pabot rather than robot. For example, -o parallel 2 will run half of the tests in one process and half in another. If not provided, all tests will run in a single process using the standard robot test runner.",
+            "type": "integer",
```

Comment:
```suggestion
```

CumulusCI does not currently do anything with this. We want to add declarative option type validation but it will be configured differently.

Reply:
The type shows up when  you do `cci task info robot`, which I thought was reason enough to include it.

Reply:
Oh huh, I didn't realize that. I guess I have no objection, but it doesn't look like we are actually using that anywhere other than the test for `doc_task`, and we'll likely need to replace it when we finish up #1618

Reply:
Ok, I'll remove it.